codespell_lib/data/dictionary.txt

```diff
@@ -3861,6 +3861,7 @@ blonged->belonged
 blonging->belonging
 blongs->belongs
 bloted->bloated
+blueish->bluish
```

Comment:
Fix the tests, this is a variant not a typo:
https://english.stackexchange.com/a/18827

Maybe move into usage or something?

Reply:
33457ad